src/lints/unwind_function_missing.ron

```diff
@@ -0,0 +1,51 @@
+SemverQuery(
+    id: "unwind_function_missing",
+    human_readable_name: "unwind function missing",
```

Comment:
I'm confused about the name of this lint. It seems like the function isn't missing, and the issue instead is that its ABI changed, right?

Reply:
Sorry for the confusion. My intention for this lint name is: the new version of code is "missing the original unwind function" as the ABI has been changed from an unwind ABI to an non-unwind ABI. 

If the lint name is misleading, would changing the lint name to pub_function_no_longer_unwind be better?

Reply:
We want to use similar names for similar lints, since that consistency will help our users find and remember related lints. This lint is most similar to the existing `function_changed_abi` lint, so perhaps something like `function_abi_no_longer_unwind`?

We historically haven't been good about whether we include `pub` or not =/ Around when #58 gets resolved I'll probably go through all the lints and remove all `pub` prefixes from lints, since most lint names don't include them and `pub` is implied for semver checks anyway.

Reply:
Also, let's rename the PR to match the new name of the lint — if you don't get a chance to do it, I can do it before merging as well.

Reply:
Sure, I just updated the lint name and the PR name